src/App.tsx

```diff
@@ -1,16 +1,26 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+const moviesList: Movie[] = moviesFromServer.map(movie => ({ ...movie }));
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesList);
```

Comment:
redundant variable 
```suggestion
  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
```
Missed type for useState